### PR TITLE
Add import geojson data first draft

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Adding docs for new Import Data (GeoJSON).

### Checklist

- [ ] Checked spelling and grammar
- [ ] Provided code samples where relevant
- [ ] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — COMMENTED** [review]
please update the block description with the one from prod, not staging (<https://marketplace.up42.dev/block/1540a0f2-a6d5-4262-944f-21117ad658be>)

**Review by teoEnache — CHANGES_REQUESTED** [review]
this page needs to be from prod: <https://marketplace.up42.dev/block/1540a0f2-a6d5-4262-944f-21117ad658be>

**Review by markusuwe — APPROVED** [review]
LGTM

File: source/up42-blocks/data/up42-import-geojson.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+.. _up42-import-geojson-block:
+
+Import Data (GeoJSON)
+=====================
+For more information, please read the `block description <https://marketplace.up42.dev/block/1540a0f2-a6d5-4262-944f-21117ad658be>`_.
+
+Block type: ``DATA``
+
+This block enables using GeoJSON vector data, stored in a bucket on *Google Cloud Storage (GCS)*
```
Comment (perusio) [inline source/up42-blocks/data/up42-import-geojson.rst:13]: Do you mean italic or bold here?

File: source/up42-blocks/data/up42-import-geojson.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+or *Amazon Web Services (AWS)*, in a workflow on UP42.
+
+Within the bucket, the user can select specific images (via the filenames) or search by
+location and time. The search can also be limited to a subfolder in the bucket via the
```
Comment (perusio) [inline source/up42-blocks/data/up42-import-geojson.rst:17]: location and date

File: source/up42-blocks/data/up42-import-geojson.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+`prefix` parameter.
+The block outputs the scene data and an automatically created `data.json` file with the scene metadata.
```
Comment (perusio) [inline source/up42-blocks/data/up42-import-geojson.rst:19]: is it not:
``data.json``?

File: source/up42-blocks/data/up42-import-geojson.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+.. tip::
+
+    In order to access the bucket, the access credentials need to be provided via :ref:`UP42 environment variables <environments-credentials-pass-through-tutorial>`.
+    For AWS, provide the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables.
```
Comment (perusio) [inline source/up42-blocks/data/up42-import-geojson.rst:24]: Ditto it requires two ``  to show up as code.

**Review by perusio — CHANGES_REQUESTED** [review]
Minor language and formatting issues below.

File: source/up42-blocks/data/up42-import-geojson.rst (hunk 1 above)
Comment (nekhtiari) [reply source/up42-blocks/data/up42-import-geojson.rst:13]: Sorry I do not understand, with one star is italic, right? I just copy-paste it from the other two custom data blocks.

File: source/up42-blocks/data/up42-import-geojson.rst (hunk 3 above)
Comment (nekhtiari) [reply source/up42-blocks/data/up42-import-geojson.rst:19]: oh sorry this is a leftover, will change it

File: source/up42-blocks/data/up42-import-geojson.rst (hunk 2 above)
Comment (nekhtiari) [reply source/up42-blocks/data/up42-import-geojson.rst:17]: done

File: source/up42-blocks/data/up42-import-geojson.rst (hunk 4 above)
Comment (nekhtiari) [reply source/up42-blocks/data/up42-import-geojson.rst]: done

**Comment by nekhtiari** [comment]
> this page needs to be from prod: https://marketplace.up42.dev/block/1540a0f2-a6d5-4262-944f-21117ad658be

Updated :)

**Review by perusio — APPROVED** [review]
LGTM.

**Pull request merged**